ps/ps.xml

```diff
@@ -0,0 +1,134 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
+
+<!-- 
+    Please see the license.html file included with this distribution for 
+    attribution and copyright information.
+-->
+
+<root>
+    <windowclass name="partysheet_host" merge="join">
+        <sheetdata>
+            <sub_ps name="domain">
+                <class>ps_domain</class>
+            </sub_ps>
+            <sub_ps name="powerpool">
+                <class>ps_powerpool</class>
+            </sub_ps>
+            <tabs_partysheet name="tabs" merge="delete" />
+            <tabs_partysheet name="tabs">
+                <tab>
+					<icon>tab_main</icon>
+					<subwindow>main</subwindow>
+				</tab>
+				<tab>
+					<icon>tab_inventory</icon>
+					<subwindow>inventory</subwindow>
+				</tab>
+                <tab>
+					<icon>tab_domain</icon>
+					<subwindow>domain</subwindow>
+				</tab>
+                <tab>
+					<icon>tab_powerpool</icon>
+					<subwindow>powerpool</subwindow>
+				</tab>
+				<tab>
+					<icon>tab_order</icon>
+					<subwindow>order</subwindow>
+				</tab>
+				<tab>
+					<icon>tab_xp</icon>
+					<subwindow>xp</subwindow>
+				</tab>
+			</tabs_partysheet>
```

Comment:
At the cost of a bit of complexity, conflict risk can be reduces via script. Either way, merge="replace" should allow a single line, as an aside.
```suggestion
			<tabs_partysheet name="tabs">
				<script>
					function onInit()
						table.insert(tab, 3, {
							subwindow = {"domain"},
							icon = {"tab_domain"}
						});
						table.insert(tab, 4, {
							subwindow = {"powerpool"},
							icon = {"tab_powerpool"}
						});
						super.onInit();
					end
				</script>
			</tabs_partysheet>
```